frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -46,82 +47,120 @@
      :east  (+ lng (/ img-x-bounds 2))
      :west  (- lng (/ img-x-bounds 2))}))
 
-;; TODO: Remove, unused
-#_(defn bounds-for-point
-  "Uses current bounds and a map point coordinate to get the map bounds centered
-   on that point. Called from get-feature-info, where we have both the geographic
-   bounds and xy map coordinates."
-  [{:keys [lat lng]}
-   {:keys [north south east west]}]
-  (let [x-bounds (Math/abs (- west east))
-        y-bounds (Math/abs (- north south))]
-    {:north (+ lat (/ y-bounds 2))
-     :south (- lat (/ y-bounds 2))
-     :east  (+ lng (/ x-bounds 2))
-     :west  (- lng (/ x-bounds 2))}))
-
-(defn get-feature-info-request
-  [request-id per-request img-size img-bounds point]
-  (let [layers->str   #(->> % (map layer-name) reverse (string/join ","))]
-   ;; http://docs.geoserver.org/stable/en/user/services/wms/reference.html#getfeatureinfo
-    (for [[[server-url info-format] active-layers] per-request
-          :let [layers   (layers->str active-layers)]]
-      (let [params {:REQUEST       "GetFeatureInfo"
-                    :LAYERS        layers
-                    :QUERY_LAYERS  layers
-                    :WIDTH         (:width img-size)
-                    :HEIGHT        (:height img-size)
-                    :BBOX          (->> img-bounds
-                                        (bounds->projected wgs84->epsg3112)
-                                        (bounds->str 3112))
-                    :FEATURE_COUNT 1000
-                    :STYLES        ""
-                    :X             50
-                    :Y             50
-                    :TRANSPARENT   true
-                    :CRS           "EPSG:3112"
-                    :SRS           "EPSG:3112"
-                    :FORMAT        "image/png"
-                    :INFO_FORMAT   info-format
-                    :SERVICE       "WMS"
-                    :VERSION       "1.1.1"}]
-        {:method          :get
-         :uri             server-url
-         :params          params
-         :response-format (ajax/text-response-format)
-         :on-success      [:map/got-featureinfo request-id point info-format]
-         :on-failure      [:map/got-featureinfo-err request-id point]}))))
-
-(defn get-feature-info [{:keys [db]} [_ {:keys [size bounds]} point]]
+(defmulti get-feature-info #(second %2))
```

Comment:
Minor point, but I often like naming the arguments in the dispatch function, just to make it clear what arguments it's receiving.

Reply:
Ah cool, makes sense. Will do so in future 👍